client.go

```diff
@@ -974,7 +980,7 @@ func (f *File) readChunkAt(ch chan result, b []byte, off int64) (n int, err erro
 		}
 	}
 
-	return
+	return len(b), nil
```

Comment:
We make the return here more precise and narrow. Since we know that reaching here these are the two values these return values logically _must_ have, we say so explicitly.